zebrad/Cargo.toml

```
@@ -6,16 +6,19 @@ authors = ["Zcash Foundation <zebra@zfnd.org>"]
 description = "A consensus-compatible Zcash node client written in Rust."
 
 [dependencies]
+tokio-core = "0.1.6"
+futures = "0.1"
+hyper = "0.12"
+tracing = "0.1"
+tracing-fmt = "0.0.1-alpha.3"
+tracing-subscriber = "0.0.1-alpha.2"
```

There is now an [0.1 release](https://crates.io/crates/tracing-subscriber/0.1.0) of this crate which should probably be used instead. 
```suggestion
tracing-subscriber = "0.1"
```